AmazonChimeSDK/AmazonChimeSDK/realtime/RealtimeControllerFacade.swift

```diff
@@ -55,4 +55,19 @@ import Foundation
     /// - Parameter lifetimeMs: Message lifetime in milisecond, 5 mins max, default 0
     /// - Throws: SendDataMessageError
     func realtimeSendDataMessage(topic: String, data: Any, lifetimeMs: Int32) throws
+
+    /// Enable/disable Voice Focus (ML-based noise suppression)
+    ///
+    /// Note: this API can only be called after audio session was started.
+    ///
+    /// - Parameter enabled: A `Bool` value, where `true` to enable; `false` to disable
+    /// - Returns: Whether the enable/disable action was successful
+    func realtimeSetVoiceFocusEnabled(enabled: Bool) -> Bool
+
+    /// Get if Voice Focus (ML-based noise suppression) is enabled or not
+    ///
+    /// Note: this API can only be called after audio session was started.
+    ///
+    /// - Returns: `true` if Voice Focus is enabled; `false` if Voice Focus is not enabled, or the audio session was not started yet
```

Comment:
Not sure if we need "Note" for this API since we explained it returns `false` if audio session is not started.

Reply:
Make sense, updated.